de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/ScriptHandler.java

```diff
@@ -156,7 +157,13 @@ public static ScriptHandler createHandler(String script_type, List<String> packa
       if (type.startsWith("r")) {
         handler = new RScriptHandler(packages);
       } else if (type.startsWith("py")) {
-        handler = new PythonScriptHandler();
+        if(type.startsWith("python 2")) {

```

Comment:
We need one white space to separate keywords like if and else. https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace